### ISSUE TITLE
How to install the extension on Windows?

### ISSUE THREAD

**Issue by mluzan** [peterjaap/magerun-addons#5]
"Installation" Instructions (provided here http://magerun.net/magerun-addons-by-peter-jaap-blaakmeer/ ) work for Linux only.
How can we install the extension on Windows?

**Comment by peterjaap** [comment]
I have no idea. It might be better to ask this question on the magerun Github page (github.com/netz98/n98-magerun), since it is magerun specific and not my addon specific.

**Comment by mluzan** [comment]
Hello!
Thank you for your reply. We were able to run the extension. But it seems that module works incorrectly.
After running this command http://screencast.com/t/IHZhxCyn and checking results, we have found, that many translations were not found. But in fact they are exactly present on our website.
For example:
The result http://screencast.com/t/t945l4MXus6
The search result in the .csv file with translations http://screencast.com/t/RuXlMzQD
